src/test/java/seedu/address/model/person/PhoneTest.java

```diff
@@ -28,14 +28,20 @@ public void isValidPhone() {
         assertFalse(Phone.isValidPhone("")); // empty string
         assertFalse(Phone.isValidPhone(" ")); // spaces only
         assertFalse(Phone.isValidPhone("91")); // less than 3 numbers
+        assertFalse(Phone.isValidPhone("+91")); // less than 3 numbers with plus
         assertFalse(Phone.isValidPhone("phone")); // non-numeric
         assertFalse(Phone.isValidPhone("9011p041")); // alphabets within digits
+        assertFalse(Phone.isValidPhone("9312-1534")); //other characters
+        assertFalse(Phone.isValidPhone("123456789012345678901234567890")); // too long phone numbers with plus sign
         assertFalse(Phone.isValidPhone("9312 1534")); // spaces within digits
```

Comment:
Shall we test some boundary values? Like a phone number with 16 numbers, and 16 numbers with a `+`?

Reply:
ok